src/components/projects/view/application-hour-tile.svelte

```diff
@@ -32,7 +32,7 @@
     try {
       const thisReport = report;
       application.reports = application.reports.filter(
-        report => thisReport.reporter_email !== account.email,
+        report => thisReport.reporter_email !== report.reporter_email,
```

Comment:
If I understand correctly, this will delete all reports by every other reporter but the current user. I believe the intention was to delete the report that was submitted by the current user, leaving reports by other moderators intact. I believe It was correct initially.

Reply:
It couldn't have been correct initially since the filter function didn't depend on the element

Reply:
Perhaps it should be something like
```suggestion
        report => thisReport !== report,
```

Reply:
Doesn't really matter, since the `reporter_email` uniquely identifies the report within an application, but okay, this might be faster

Reply:
Or even just a simple `indexOf` (or `findIndex`) combined with `splice` should do the trick